usps/main.py

```diff
@@ -0,0 +1,112 @@
+"""
+USPS Microservice FastAPI Web App.
+"""
+
+import json
+import logging
+import ssl
+import base64
+from typing import Optional
+from uuid import UUID
+from http import HTTPStatus
+from fastapi import FastAPI, Header
+from fastapi.responses import JSONResponse
+from aiohttp import ClientSession, ClientError
+from pydantic import BaseModel
+from google.oauth2 import service_account
+from google.auth.transport.requests import Request
+from usps.config import settings
+from usps import transaction_log
+
+app = FastAPI()
+
+
+class AddressVerificationInfo(BaseModel):
+    """
+    Request model used for generating documentation and input validation.
+    """
+
+    first_name: str
+    last_name: str
+    middle_name: Optional[str] = None
+    suffix: Optional[str] = None
+    delivery_address: str
+    address_city_state_zip: str
+
+
+class VerifiedResponse(BaseModel):
+    """
+    Response model returning a success used for generating documentation.
+    """
+
+    uid: UUID
+    confidence_indicator: str
+
+
+class ErrorResponse(BaseModel):
+    """
+    Response model returning a failure used for generating documentation.
```

Comment:
This would be clearer as `Response model for failed (non-2XX) responses from the USPS API`